renderdoc/driver/gl/gl_common.cpp

```diff
@@ -794,6 +794,11 @@ bool GLInitParams::IsSupportedVersion(uint64_t ver)
   if(ver == CurrentVersion)
     return true;
 
+  // 0x1A -> 0x1B - supported MSAA and Multiview framebuffer attachments, which added
```

Comment:
Shouldn't this code block go under
```// we can check other older versions we support here.```
comment and not above it? It seems to do exactly what the comment says.

Reply:
In other cases I've removed the comment once I added at least one older version, since it's no longer needed as placeholder. It's a minor thing I was just going to clean up later, too nitpicky even for me to quibble in a review :stuck_out_tongue:.